WordPressEditor/build.gradle

```diff
@@ -41,6 +44,17 @@ dependencies {
     compile 'org.wordpress:analytics:1.0.0'
     releaseCompile project(path:':libs:utils:WordPressUtils', configuration: 'release')
     debugCompile project(path:':libs:utils:WordPressUtils', configuration: 'debug')
+
+    // Test libraries
+    testCompile 'junit:junit:4.10'
+    testCompile 'org.mockito:mockito-core:1.10.19'
+    testCompile 'org.robolectric:robolectric:2.4'
```

Comment:
This version is not compatible with `appCompat` but probably irrelevant for now. Any possibility where `appCompat` comes into play in the project?